Tests/TuistCacheTests/ContentHashing/SourceFilesContentHasherTests.swift

```diff
@@ -0,0 +1,64 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistCoreTesting
+import XCTest
+import TuistSupport
+import TuistCacheTesting
+@testable import TuistCache
+@testable import TuistSupportTesting
+
+final class SourceFilesContentHasherTests: TuistUnitTestCase {
+    private var subject: SourceFilesContentHasher!
+    private var mockContentHasher: MockContentHashing!
+    private var temporaryDirectory: TemporaryDirectory!
+    private let sourceFile1Path = AbsolutePath("/file1")
```

Comment:
I'd do what you have done with the other attributes, initializing them in the `setUp` and setting them to nil in the `tearDown` method. Not a big deal, but if we do many of these, we'll be using unnecessary memory.

Reply:
AFAIK, having a `- setUp` that always sets a var to the same value, is almost equivalent of declaring that variable as a constant, as they will both have the same value for each test and be released after the test case finshes.